src/components/paintings-page/painting-tile/painting-tile.jsx

```diff
@@ -4,24 +4,35 @@ import Link from "next/link";
 import styles from "./painting-tile.module.scss";
 
 export default function PaintingTile(props) {
-  const { name, imageCollection, price } = props;
+  const { name, imageCollection, price, classNames = '' } = props;
   const firstImage = imageCollection.items[0];
+
   return (
     <Link
       href={{
         pathname: `/painting/${name}`,
       }}
       className={styles.paintingTile}
     >
-      <div className={styles.imageContainer}>
+      <div
+        className={
+          classNames ? classNames.recommendedContainer : styles.imageContainer
```

Comment:
mm I dont love this. It's not a good habit to build in logic to a re-useable component that is very specific to a certain use-case. If we were to continue adding to this website over time, this classlist could hypothetically look like this:'
```
className={
  classNames.reccommendedContainer ? classNames.reccommendedContainer : classNames.newComponetContainer ? classNames.newComponetContainer : classNames.anotherNewComponetContainer ? classNames.anotherNewComponetContainer : styles.imageContainer
}
```
^ That's not pretty and not scaleable. I'd suggest finding a more scaleable solution to this, and let me know if you need help.